docs/ensembl-help/using-ensembl/getting-ensembl-data/tools-and-apis/refget/about-refget.md

```diff
@@ -0,0 +1,375 @@
+
+## What is refget?
+
+Refget is a specification which defines a unified method to access reference sequences using an identifier system derived from the sequence itself. This identifier is normally a checksum such as hex encoded `MD5` or a scheme called [`sha512t24u`](https://samtools.github.io/hts-specs/refget.html#refget-checksum-algorithm).
+
+Refget supports the retrieval of full length sequences, a range within a sequence and metadata concerning a sequence. All of these functions are supported through the provision of the checksum identifier. For example to retrieve the first 100 base pairs from T2T-CHM13v2.0 chromosome 1 (INSDC accession: `CP068277.2`, MD5 checksum: `e469247288ceb332aee524caec92bb22`, GA4GH identifier `sha512t24u`: `ga4gh:SQ.S9dv70MOE6RecB0OYnc32EilaGTDWca-`).
+
+```bash
+$ curl -H'Accept: text/plain' 'https://beta.ensembl.org/data/refget/sequence/e469247288ceb332aee524caec92bb22?start=0&end=100'
+CACCCTAAACCCTAACCCCTAACCCTAACCCTAACCCTAACCCTAACCCTAACCCCTAAACCCTAACCCTAACCCTAACCCTAACC
+```
+
+**N.B. If you are recieving a 406 error and using a web browser to execute sequence retrieval refget URLs, try adding `accept=text/plain` to your URL parameters.**
+
+To retrieve metadata about a sequence (such as length and other known identifiers) you can use the metadata endpoint.
+
+```bash
+$ curl -H'Accept: application/json' 'https://beta.ensembl.org/data/refget/sequence/e469247288ceb332aee524caec92bb22/,metadata'
+{"metadata": {"aliases": [{"alias": "ga4gh:SQ.S9dv70MOE6RecB0OYnc32EilaGTDWca-", "naming_authority": "ga4gh"}, {"alias": "1", "naming_authority": "Ensembl"}], "length": 248387328, "md5": "e469247288ceb332aee524caec92bb22", "trunc512": "4bd76fef430e13a45e701d0e627737d848a56864c359c6be"}}
+```
+
+## Why is Ensembl using refget
+
+Refget is an approved standard by the [Global Alliance for Genomics and Health](https://www.ga4gh.org/product/refget/). Key to the system is the identifier created from sequence content, meaning that sequence retrieval is unambigous and ensures the correct sequence is always retrieved. It is currently not possible to resolve sequences within our GraphQL service as sequence length can be variable and create non-uniform large payloads. Instead clients should choose to resolve sequences individually.
+
+## Refget documentation
+
+Ensembl currently implements a refget v1.0.1 compatible service. Documentation is available from GA4GH in [text format](https://github.com/samtools/hts-specs/blob/fde0b700d7d1aeb4d40cc27c820965a6f365f650/refget.md) and [OpenAPI v3.0](https://raw.githubusercontent.com/samtools/hts-specs/fde0b700d7d1aeb4d40cc27c820965a6f365f650/pub/refget-openapi.yaml) ([including visualisation](https://rest.wiki/?https://raw.githubusercontent.com/samtools/hts-specs/fde0b700d7d1aeb4d40cc27c820965a6f365f650/pub/refget-openapi.yaml)).
```

Comment:
What do you mean by visualisation? A swagger-like playground? If so, is visualisation a usual name for it?

Reply:
Yeah. What's the right name for that?

Reply:
Interactive API documentation I think